common/keyboard.go

```diff
@@ -253,6 +251,36 @@ func (k *Keyboard) modifierBitFromKeyName(key string) int64 {
 	return 0
 }
 
+func (k *Keyboard) comboPress(keys string, opts *KeyboardOptions) error {
+	if opts.Delay != 0 {
+		t := time.NewTimer(time.Duration(opts.Delay) * time.Millisecond)
+		select {
+		case <-k.ctx.Done():
+			t.Stop()
+		case <-t.C:
+		}
+	}
+
+	kk := []string{"+"}
+	if keys != "+" {
```

Comment:
I couldn't understand what is this for? Should it be `strings.Contains`?

Reply:
Yeah, this is an interesting edge case, so in playwright:

1. `press("+")` -> types `+`
2. `press("++")` -> errors with `Unknown key: ""`
3. `press("+++")` -> types `++`
4. `press("Sift++")` -> types `+`

I'm trying to code the same behaviour, any ideas how this could be done?

Reply:
Yeah it is the same for PW, here's the script I used to test this:

```js
// @ts-check
const { test, chromium } = require('@playwright/test');

test.describe('Editing', () => {
  test('Type login creds', async () => {
    const browser = await chromium.launch({ headless: false, slowMo: 500 });
    const page = await browser.newPage();
    await page.goto('https://www.bing.com/');
    var elm = await page.$('input[name="q"]');

    await elm.press("+++"); // types in ++
    await elm.press("+"); // types in +
    await elm.press("++"); // will error and exit with `Unknown key: ""`

    await new Promise(resolve => setTimeout(resolve, 5000));

    await browser.close();
  });
});
```

Reply:
Interesting, thanks!